.github/workflows/CICD.yml

```diff
@@ -380,6 +380,7 @@ jobs:
 
   coverage:
     name: Code Coverage
+    if: "false"
```

Comment:
Can we maybe add a comment with a reference to this PR?

Reply:
Good idea. Done!